### ISSUE TITLE
Wrong Offset [2E0BA] Wrong

### ISSUE THREAD

**Issue by yoki22** [keylase/nvidia-patch#239]
**Please provide details about your environment (please complete the following information)**
- Nvidia driver version: [ 442.50]
- GPU Model: [ GTX 1060 6 GB]
- OS: [Windows 10 Pro 64]
- [x] I'm willing to participate in testing of new patch version

I get the attached offset error when trying to apply nvencodeapi64.1337 to nvEncodeAPI64.dll. I tried restarting my machine and disabling plex but still get the same issue.

Let me know if I need to provide additional information.

![Capture](https://user-images.githubusercontent.com/61902252/76140305-f2fe5780-6026-11ea-89a3-ef84790b3dbc.JPG)

**Comment by Snawoot** [comment]
Hello,

Could you please attach nvencodeapi64.1337 file you used?

**Comment by Snawoot** [comment]
Also please attach your nvEncodeAPI64.dll and nvEncodeAPI.dll files and check you really have driver version 442.50.

**Comment by yoki22** [comment]
Hi 

Thanks for the quick response. Your last suggestion was actually my issue. For some reason, even though I ran the driver package linked on your site for the 442.50 drivers Windows was still installing the 432.00 drivers instead. Also, when installing nvidia control panel from the windows store it also downgrades the drivers to 432.00. Once I did a clean driver install it patched without issue.

Thanks again. Great work!
